k8s/perftest/common/sscs/sscs-tya-notif.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: sscs-tya-notif
```

Comment:
Why tya app? I think it should be core-front end

Reply:
this is notification service